vc_agent

```diff
@@ -10,6 +17,8 @@ export VC_AGENT_INSTALL_DIR=$VC_HOME/bin
 export VC_DRV_MANUAL_HOST_URI="$DATABASE_URL"
 export VC_HOSTNAME="${VC_HOSTNAME:-$(hostname)}"
 
+[ -f $VC_HOME/.aws/credentials ] || make_credentials
```

Comment:
Might be worth making this check that the variables are set, otherwise not making this file.  Also worth some output if that happens.